ls1intum/Artemis
# `Exam mode`: Fix breadcrumbs for import of exam exercises

### Checklist
#### General


(https://docs.artemis.cit.tum.de/dev/guidelines/server/#rest-endpoint-best-practices-for-authorization) 
#### Client
### Motivation and Context
While enabling the import of file upload exercises for exams, I noticed that there was a duplicated import breadcrumb when navigating to the import page.
When clicking the the second breadcrumb, it resulted in a Bad Request.
For programming exercises, this breadcrumb is already hidden. This PR adds it for the other exercise types.



### Description
Extended the conditions when to **not** show an breadcrumb for exam exercise imports
### Steps for Testing

Prerequisites:
- 1 Editor or Instructor
- One exercise that can be imported.

1. Log in to Artemis
2. Navigate to Course Administration
3. Import an exercise and see that there is now only one import breadcrumb
4. Navigating using the breadcrumbs should still work as expected

#### Exam Mode Testing
Testing this PR equals testing the exam mode as it only affects the exam mode.

### Review Progress






#### Code Review
#### Manual Tests
#### Exam Mode Test

### Test Coverage





### Screenshots


Before
![breadcrumbs_before](https://user-images.githubusercontent.com/84102468/221869918-6bd44784-e665-409b-8b38-8247906e43f0.png)

After
![breadcrumbs_after](https://user-images.githubusercontent.com/84102468/221869990-f697d1ed-2ce0-4c67-b499-952ce86313c7.png)
